docs/configuration.rst

```diff
@@ -79,6 +79,10 @@ You may want to pull/push images from/to a custom docker registry. For instance,
 Open edX customisation
 ~~~~~~~~~~~~~~~~~~~~~~
 
+- ``OPENEDX_PLATFORM_REPOSITORY`` (default: ``"https://github.com/openedx/edx-platform.git"``)
```

Comment:
Can we rename this to `EDX_PLATFORM_REPOSITORY`? `OPENEDX_*` settings refer to values that are used across multiple Open edX apps.

Reply:
I figured `OPENEDX_*` settings would also apply to anything that's relevant to building the `openedx` image, but sure I can rename it.

Reply:
Done.